Cargo.toml

```diff
@@ -338,7 +338,7 @@ codegen-units = 4
 
 [profile.test]
 opt-level = 0
-debug = 0
+debug = 2
```

Comment:
why change it

Reply:
This was for testing and debugging, it sneaked in by mistake.

Reply:
You can debug the test if you do the same change.😉